aspnetcore/mvc/controllers/bind-tryparse/7.0-samples/BindUsingTryParse/BindTryParseAPI/Models/Culture.cs

```diff
@@ -0,0 +1,42 @@
+using System.Globalization;
+
+namespace BindTryParseAPI.Models
+{
+    // <snippet>
+    public class Culture : IParsable<Culture>
```

Comment:
Could it just be `Culture: CultureInfo, IParsable<Culture>`, so you don't need to have a `CultureInfo` property?